### ISSUE TITLE
Jailbreak (latest version) Hanging?

### ISSUE THREAD

**Issue by richard-canning** [NSSpiral/Blackb0x#73]
Hi,

I have the ATV 3.2 (A1469 model) and downloaded the latest version of Blackb0x from here (0.6.2), and installed on my mac Big Sur OS (running in Oracle VirtualBox VMWare). When I launch Blackb0x, it opens and shows my apple tv model correctly, but there are 2 greyed out labels/buttons underneath the apple TV logo. If I hover over the second "greyed out" button and click down on my mouse....a green install jailbreak button appears....I then clicked that...and it takes me to another popup screen that just has a grey cancel button on it, and seems to indicate that it is installing, but it is just hanging there and going on forever (see screenshot below). It never gets to the instructions to put the ATV into DFU mode. Any ideas? I did try a fresh restore from factory settings on my apple tv before attempting this jailbreak.

![image](https://user-images.githubusercontent.com/23638755/147306961-587efdc0-c7c5-4113-a434-c483abc43fc2.png)

Thanks,
Richard

**Comment by NSSpiral** [comment]
I'm not sure why it looks like that but the popup window 2 is the window to help you enter DFU. If you just put it in DFU it should continue.

Hold Menu + Down until LED starts flashing, then Menu + Play/Pause until it starts rapidly flashing.

**Comment by richard-canning** [comment]
Hi,

Thank you for your prompt reply. I just tried putting into DFU mode first before trying to install according to your instructions and light is rapidly flashing and macOS is telling me the ATV is in DFU. I left Blackb0x open while going into DFU. Did notice after it picked up that the ATV is in DFU, that there was an additional button showed up under the install button called "Tethered Boot". Not sure what that means, but I didn't click on that one. I then clicked on the Install Jailbreak button and then the 2 buttons disappeared. Does that mean it is installing? Does it take a really time to install? The lettering in the label is black, so I can't read what it says.

![image](https://user-images.githubusercontent.com/23638755/147391982-9fa87c20-f1bf-4b05-8a7b-8c9d63d69774.png)

Thanks again for your help and happy holidays.

Cheers,
Richard
